modules/remotecontrol/Receiver.js

```diff
@@ -7,20 +10,36 @@ const ConferenceEvents = JitsiMeetJS.events.conference;
  * API module. From there the events can be received from wrapper application
  * and executed.
  */
-class Receiver {
+export default class Receiver {
     /**
      * Creates new instance.
      * @constructor
      */
-    constructor() {}
+    constructor() {
+        this.enabled = false;
+    }
+
+    /**
+     * Enables / Disables the remote control
+     * @param {boolean} enabled the new state.
+     */
+    enable(enabled) {
+        if(this.enabled !== enabled && enabled === true) {
+            this.enabled = enabled;
+            // Announce remote control support.
+            APP.connection.addFeature(DISCO_REMOTE_CONTROL_FEATURE, true);
+        }
+    }
 
     /**
      * Attaches listener for ConferenceEvents.ENDPOINT_MESSAGE_RECEIVED events.
      */
     start() {
-        APP.conference.addConferenceListener(
-            ConferenceEvents.ENDPOINT_MESSAGE_RECEIVED,
-            this._onRemoteControlEvent);
+        if(this.enabled) {
```

Comment:
Same as similar comments above. I find it strange to allow calling start when Receiver is not enabled without logging or throwing any errors.